cameo/strain_design/heuristic/evolutionary/objective_functions.py

```diff
@@ -214,35 +233,35 @@ class biomass_product_coupled_min_yield(YieldFunction):
 
     """
 
-    def __init__(self, biomass, product, substrate, *args, **kwargs):
-        super(biomass_product_coupled_min_yield, self).__init__(product, substrate, *args, **kwargs)
-        if isinstance(biomass, Reaction):
-            biomass = biomass.id
-        self.biomass = biomass
-
     def __call__(self, model, solution, targets):
-        try:
-            biomass_flux = round(solution.fluxes[self.biomass], config.ndecimals)
-            fva_res = flux_variability_analysis(model, reactions=[self.product], fraction_of_optimum=1)
-            min_product_flux = fva_res["lower_bound"][self.product]
-
-            substrate_flux = round(abs(solution.fluxes[self.substrate]), config.ndecimals)
-            return round((biomass_flux * min_product_flux) / substrate_flux, config.ndecimals)
+        biomass_flux = round(solution.fluxes[self.biomass], config.ndecimals)
+        fva_res = flux_variability_analysis(model, reactions=[self.product], fraction_of_optimum=1)
```

Comment:
This will solve the model twice (min and max). It could be made faster by manually setting the objective and only solving the minimization